bin/kano-world-launcher

```diff
@@ -60,15 +60,15 @@ class chromium_browser:
                 self._modify_local_state()
             except Exception:
                 self._create_local_state()
-            else:
```

Comment:
The typo was here: the else belonged to the `try/except` which prevented the `create` method from being called the first time you launched KW